api/src/prescription/campaign-participation/domain/models/CampaignToStartParticipation.js

```diff
@@ -17,12 +18,12 @@ class CampaignToStartParticipation {
     this.type = type;
     this.idPixLabel = idPixLabel;
     this.archivedAt = archivedAt;
-    this.isRestricted = isRestricted;
     this.multipleSendings = multipleSendings;
     this.assessmentMethod = assessmentMethod;
     this.skillCount = skillCount;
     this.organizationId = organizationId;
     this.deletedAt = deletedAt;
+    this.#computeisRestrictedAccess(isManagingStudents, hasLearnersImportFeature);
```

Comment:
**to do :**
- appliquer la même modification que pour `CampaignToJoin`
- regarder la différence entre `CampaignToJoin` et `CampaignToStartParticipation` pour une dette tech potentielle